packages/commerce/backend/src/open-payments/service.ts

```diff
@@ -59,8 +59,10 @@ export class OpenPayments implements IOpenPayments {
       .get({
         url: paymentPointerUrl
       })
-      .catch((err) => {
-        this.logger.error(err)
+      .catch(() => {
```

Comment:
`preparePayment`  method is way to big, can it be divided somehow?

Reply:
Around 100 LOC seems reasonable to me. I don't see how we can abstract more methods. For example, I don't think moving `this.opClient.incomingPayment.create` (around 20 LOC) into a separate method would make a difference.